saleor/graphql/order/mutations/orders.py

```diff
@@ -677,16 +678,14 @@ def get_instance(cls, info, **data):
         return instance
 
     @classmethod
-    @traced_atomic_transaction()
-    def perform_mutation(cls, root, info, **data):
-        order = cls.get_instance(info, **data)
-        order.status = OrderStatus.UNFULFILLED
-        order.save(update_fields=["status"])
-        payment = order.get_last_payment()
-        manager = info.context.plugins
+    def process_payment(cls, order, payment, info, manager, to_pay):
         if payment and payment.is_authorized and payment.can_capture():
+            amount = min(to_pay, Money(payment.get_charge_amount(), payment.currency))
             gateway.capture(
-                payment, info.context.plugins, channel_slug=order.channel.slug
+                payment,
+                info.context.plugins,
+                channel_slug=order.channel.slug,
+                amount=amount,
             )
             order_captured(
```

Comment:
Proably this wasn't included in DoD, but this should be raised only once for mutation, not for all payments. It is an event for capturing order.

Reply:
Hm. as I image, it was the result of many payment instead one in this mutation. It uses payment to send event that payment captured order. Now there could be may payments. Which one should I used to sent event?

Reply:
We should do that in the same way as we're going to do for Order refund action - https://app.clickup.com/t/u9ejzr

Reply:
OK